src/spatial_ref/srs.rs

```diff
@@ -65,19 +65,25 @@ impl PartialEq for SpatialRef {
 }
 
 impl SpatialRef {
-    pub fn new() -> SpatialRef {
+    pub fn new(definition: &str) -> Result<SpatialRef> {
```

Comment:
You're right, there is use-cases for `new()` without parameters (I did this as a shortcut as currently the other exposed functions only allow to "import" a SRS, but it might be better to keep a method to create an empty Spatial Reference).

As you advised, it keeps now a `new()` method to create a spatial reference without parameters and a `new_from_definition()` using `OSRSetFromUserInput`.